src/telegraf.ts

```diff
@@ -1,33 +1,34 @@
-/* eslint @typescript-eslint/no-var-requires: "warn" */
-import * as http from 'http'
-import * as https from 'https'
 import type * as tt from '../typings/telegram-types'
 import type ApiClient from './core/network/client'
+import BaseScene from './scenes/base'
 import Composer from './composer'
 import Context from './context'
 import crypto from 'crypto'
+import Debug from 'debug'
+import Extra from './extra'
 import generateCallback from './core/network/webhook'
+import http from 'http'
+import https from 'https'
```

Comment:
What's the benefit of doing this? The drawback is obvious -- it'll need to be changed back if `esModuleInterop` is disabled.

Reply:
I prefer to make it as tidy as possible for now, esModuleInterop changes later are easy to fix